listView.js

```diff
@@ -96,14 +106,15 @@ class EnhancedListView extends Component {
     }
   }
   render() {
-    return <ListView
-      scrollRenderAheadDistance={RENDER_AHEAD_DISTANCE}
-      scrollEventThrottle={1}
-      {...this.props}
-      onScroll={this.onScroll.bind(this)}
-      ref={component => this.listView = component}
-      renderRow={this.renderRow.bind(this)}
-      renderSectionHeader={this.renderSectionHeader.bind(this)}/>
+    return (
+      <ListView
+        {...this.props}
```

Comment:
If we were to re-order the props to have the spread props at the end, it would allow us to manually override the props that we are hard-coding in this library.

For example, an ordering like below would allow users to override the `scrollEventThrottle` property if they wanted to throttle this component.
```js
      <ListView
        ref={component => this.listView = component}
        onScroll={this.onScroll.bind(this)}
        renderRow={this.renderRow.bind(this)}
        renderSectionHeader={this.renderSectionHeader.bind(this)}
        {...this.props}
      />
```